charts/blackduck-connector/templates/opssight-scanner.yaml

```diff
@@ -0,0 +1,167 @@
+  namespace: {{ .Release.Namespace }}
+spec:
+  {{- if eq .Values.status "Running" }}
+  replicas: {{ .Values.scanner.replicas }}
+  {{- else }}
+  replicas: 0
+  {{- end }}
+  selector:
+    matchLabels:
+      app: opssight
+      component: scanner
+      name: {{ .Release.Name }}
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        app: opssight
+        component: scanner
+        name: {{ .Release.Name }}
+      name: {{ .Release.Name }}-opssight-scanner
+    spec:
+      containers:
+      - args:
+        - /etc/scanner/opssight.json
+        command:
+        - ./opssight-scanner
+        envFrom:
+        - secretRef:
+            name: {{ .Release.Name }}-opssight-blackduck
+        {{- if .Values.scanner.registry }}
+          {{- if .Values.scanner.imageTag }}
+        image: {{ .Values.scanner.registry }}/opssight-scanner:{{ .Values.scanner.imageTag }}
+          {{- else }}
+        image: {{ .Values.scanner.registry }}/opssight-scanner:{{ .Values.imageTag }}
+          {{- end}}
+        {{- else }}
+          {{- if .Values.scanner.imageTag }}
+        image: {{ .Values.registry }}/opssight-scanner:{{ .Values.scanner.imageTag }}
+          {{- else }}
+        image: {{ .Values.registry }}/opssight-scanner:{{ .Values.imageTag }}
+          {{- end}}
+        {{- end}}
+        name: scanner
+        ports:
+        - containerPort: {{ .Values.scanner.port }}
+          protocol: TCP
+        resources:
+          {{- toYaml .Values.scanner.resources | nindent 12 }}
+        securityContext:
+          privileged: false
+        volumeMounts:
+        - mountPath: /etc/scanner
+          name: scanner
+        - mountPath: /var/images
+          name: var-images
+      - args:
+        - /etc/image-getter/opssight.json
+        command:
+        - ./opssight-image-getter
+        envFrom:
+        - secretRef:
+            name: {{ .Release.Name }}-opssight-blackduck
+        {{- if .Values.imageGetter.registry }}
+          {{- if .Values.imageGetter.imageTag }}
+        image: {{ .Values.imageGetter.registry }}/opssight-image-getter:{{ .Values.imageGetter.imageTag }}
+          {{- else }}
+        image: {{ .Values.imageGetter.registry }}/opssight-image-getter:{{ .Values.imageTag }}
+          {{- end}}
+        {{- else }}
+          {{- if .Values.imageGetter.imageTag }}
+        image: {{ .Values.registry }}/opssight-image-getter:{{ .Values.imageGetter.imageTag }}
+          {{- else }}
+        image: {{ .Values.registry }}/opssight-image-getter:{{ .Values.imageTag }}
+          {{- end}}
+        {{- end}}
+        name: image-getter
+        ports:
+        - containerPort: {{ .Values.imageGetter.port }}
+          protocol: TCP
+        resources:
+          {{- toYaml .Values.imageGetter.resources | nindent 12 }}
+        securityContext:
+          privileged: false
+        volumeMounts:
+        - mountPath: /etc/image-getter
+          name: image-getter
+        - mountPath: /var/images
```

Comment:
does the mount path is from .Values.scanner.imageDirectory?

Reply:
done